packages/block-library/src/group/block.json

```diff
@@ -17,6 +17,10 @@
 			"enum": [ "all", "insert", false ]
 		}
 	},
+	"providesContext": {
+		"layout": "layout",
+		"orientation": "orientation"
```

Comment:
The block doesn't have an `orientation` attribute.

Reply:
Thanks! I've had a go in https://github.com/WordPress/gutenberg/pull/39743/commits/3ac8f3c434e11afe436099cb3ad1ae9dc2aed784 at moving the `providesContext` to the layout block support instead. I think a good case could be made for passing it down as context in all cases where the layout support is opted into, particularly given that child blocks still need to explicitly opt-in to allow accessing that context.